### ISSUE TITLE
Process silently fails when the driver times out

### ISSUE THREAD

**Issue by elsassph** [mikestead/lighthouse-batch#16]
```
Runtime error encountered: Waiting for DevTools protocol response has exceeded the allotted time. (Method: Runtime.evaluate)
LHError: PROTOCOL_TIMEOUT
    at Timeout.setTimeout [as _onTimeout] (/path/to/job/node_modules/lighthouse/lighthouse-core/gather/driver.js:291:21)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
1/1: Lighthouse analysis FAILED for https://dev.domain.ext
```
Yet the process returns a success code.

**Comment by elsassph** [comment]
Running Lighthouse in "headless mode" seem to fix the isse - see PR #19.

**Comment by elsassph** [comment]
@mikestead the headless mode fixes our issue but we still have IMHO a problem that `lighthouse-batch` doesn't exit with an error if it's not able to run some tests. At least it could be an option.
